### PR TITLE
Added example to Colab installation documentation

### PR DESCRIPTION
**Pull request by Iced-Tea3**
## Motivation

A recently third-party blog made me realize the docs tutorial of Colab wasn't as clear as it can be. An example to run might be easier to follow for Colab beginners.
## Explanation for Changes

[Documentation Reference](https://manimce--1659.org.readthedocs.build/en/1659/installation/colab.html)

## Checklist
- [x] I have read the [Contributing Guidelines](https://docs.manim.community/en/latest/contributing.html)
- [x] I have written a descriptive PR title (see top of PR template for examples)
- [x] I have written a changelog entry for the PR or deem it unnecessary
- [ ] My new functions/classes either have a docstring or are private
- [ ] My new functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [ ] My new documentation builds, looks correctly formatted, and adds no additional build warnings

## Reviewer Checklist
- [ ] The PR title is descriptive enough
- [ ] The PR is labeled correctly
- [ ] The changelog entry is completed if necessary
- [ ] Newly added functions/classes either have a docstring or are private
- [ ] Newly added functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [ ] Newly added documentation builds, looks correctly formatted, and adds no additional build warnings

### CODE REVIEW

**Review by Darylgolden — CHANGES_REQUESTED** [review]
You'll also need to include a cell to import manim (from manim import *)

**Comment by Iced-Tea3** [comment]
> You'll also need to include a cell to import manim (from manim import *)

Ah yes, i'll add that

File: docs/source/installation/colab.rst [hunk 1]
```diff
@@ -10,4 +10,22 @@ You can install Colab by running the following in a cell:
    !pip install manim
    !pip install IPython --upgrade
 
-After the execution has completed, you will be prompted to restart the runtime. Click the "restart runtime" button at the bottom of the cell output.
\ No newline at end of file
+After the execution has completed, you will be prompted to restart the runtime. Click the "restart runtime" button at the bottom of the cell output.
+
+In another cell, add a manim import.
```
Comment (Darylgolden) [inline docs/source/installation/colab.rst:15]:
```suggestion
In the next cell, import manim like so:
```

**Review by Darylgolden — CHANGES_REQUESTED** [review]

**Comment by Nikhil0504** [comment]
you can also link this https://github.com/ManimCommunity/manim/blob/main/example_scenes/manim_jupyter_example.ipynb to show how it will be.

**Comment by Iced-Tea3** [comment]
> you can also link this https://github.com/ManimCommunity/manim/blob/main/example_scenes/manim_jupyter_example.ipynb to show how it will be.

I'm not sure how to do that 😅 do you mean just adding the link?

**Comment by Darylgolden** [comment]
> 
> 
> > you can also link this https://github.com/ManimCommunity/manim/blob/main/example_scenes/manim_jupyter_example.ipynb to show how it will be.
> 
> I'm not sure how to do that 😅 do you mean just adding the link?

I think it's fine to just leave it for now until we get a proper Jupyter tutorial on the docs.

**Review by Darylgolden — APPROVED** [review]
LGTM

**Pull request merged**